### ISSUE TITLE
[BUG] Loader crashes if vid and pid lack 0x prefix

### ISSUE THREAD

**Issue by badger200** [lotuspar/usbpty#9]
It would be nice if it worked with `-v 1234 -p abcd`, currently it crashes with zero clue as to what the issue is, only by some miracle did I discover the fix is to use `-v 0x1234 -p 0xabcd`

**Comment by giodotblue** [comment]
yeah sorry - pretty sure thats something to do with argparse, I'll take a look when I have time
